### ISSUE TITLE
Using RTags with Tramp

### ISSUE THREAD

**Issue by chughtapan** [Andersbakken/rtags#1240]
**Is your feature request related to a problem? Please describe.**
My code is on a remote VM and all the development happens on that machine. I currently ssh into the machine and open up emacs and use RTags. RTags does not work if I open the project using TRAMP.

**Describe the solution you'd like**
I would like to use RTags in the above-mentioned scenario. I tried setting rtags-tramp-enabled and adding the path to my tramp-remote-path. However, I get an error saying that the server cannot be started.

**Additional context**
I looked into the code a bit. `rtags-executable-find` returns just the executable name if tramp is enabled [here](https://github.com/Andersbakken/rtags/blob/d388a00f54c67b09af93ca0cc98c35433ebe7910/src/rtags.el#L1147) . However, when starting the server  - in `rtags-start-server-unless-running` the check fails [here](https://github.com/Andersbakken/rtags/blob/d388a00f54c67b09af93ca0cc98c35433ebe7910/src/rtags.el#L4608)

Unfortunately, I don't have significant knowledge of elisp. If someone can guide me a bit around this, I can write the fixes for this.

Thanks,
Tapan

**Comment by vibrys** [comment]
is the directory where rdm resides on remote machine added to tramp-remote-path variable?

**Comment by chughtapan** [comment]
Yes, as I mentioned above, I installed rtags on the remote machine and added the path to tramp-remote-path variable

**Comment by chughtapan** [comment]
Hi,
Sorry for the late reply. 
Here's what I can reproduce - 
0. Installed the latest version of rtags on my local machine
1. Started rdm on the remote machine and indexed the project there 
2. Opened a remote file using tramp. Tried running rc --help and verified it works. I also ran rc --dump-compile-commands as a shell command from the remote buffer and it works too

However, I still cannot browse through my code. I tried using rtags-find-symbol or rtags-find-references, I just get "RTags:  No results"
